rest/routing.go

```diff
@@ -373,6 +373,9 @@ func createDiagnosticRouter(sc *ServerContext) *mux.Router {
 	keyspace := r.PathPrefix("/{keyspace:" + dbRegex + "}/").Subrouter()
 	keyspace.StrictSlash(true)
 	keyspace.Handle("/{docid:"+docRegex+"}/_all_channels", makeHandler(sc, adminPrivs, []Permission{PermReadAppData}, nil, (*handler).handleGetDocChannels)).Methods("GET")
+	keyspace.Handle("/_sync", makeHandler(sc, adminPrivs, []Permission{PermReadAppData}, nil, (*handler).handleSyncFnDryRun)).Methods("GET")
+	keyspace.Handle("/_import_filter", makeHandler(sc, adminPrivs, []Permission{PermReadAppData}, nil, (*handler).handleImportFilterDryRun)).Methods("GET")
```

Comment:
I went back to the spec (after looking at your changes from the last review), and it looks like this endpoint is supposed to support **either** a docID on the path, or providing the body directly.  I think that still makes sense - did you decide to change that intentionally?

Reply:
The same applies to _sync - the spec looks like we want to support either docID or a provided body. In this case it might be the spec that needs to be updated, but what do you think the ideal behaviour is for this one?

Reply:
I was going to update the spec once this is merged but if the body is provided I dont think theres a need for a doc id